invenio/utils/connector.py

```diff
@@ -17,640 +17,8 @@
-            if tag not in self.cur_record:
-                self.cur_record[tag] = self.cur_controlfield
-            self.in_controlfield = True
-
-        elif name == "datafield":
-            tag = attributes["tag"]
-            self.cur_tag = tag
-            ind1 = attributes["ind1"]
-            if ind1 == " ": ind1 = "_"
-            ind2 = attributes["ind2"]
-            if ind2 == " ": ind2 = "_"
-            if tag + ind1 + ind2 not in self.cur_record:
-                self.cur_record[tag + ind1 + ind2] = []
-            self.cur_datafield = {}
-            self.cur_record[tag + ind1 + ind2].append(self.cur_datafield)
-            self.in_datafield = True
-
-        elif name == "subfield":
-            subcode = attributes["code"]
-            if subcode not in self.cur_datafield:
-                self.cur_subfield = []
-                self.cur_datafield[subcode] = self.cur_subfield
-            else:
-                self.cur_subfield = self.cur_datafield[subcode]
-            self.in_subfield = True
-
-    def characters(self, data):
-        if self.in_subfield:
-            self.buffer += data
-        elif self.in_controlfield:
-            self.buffer += data
-        elif "Search-Engine-Total-Number-Of-Results:" in data:
-            print(data)
-            match_obj = re.search("\d+", data)
-            if match_obj:
-                print(int(match_obj.group()))
-                self.counts = int(match_obj.group())
-
-    def endElement(self, name):
-        if name == "record":
-            self.in_record = False
-        elif name == "controlfield":
-            if self.cur_tag == "001":
-                self.recid = int(self.buffer)
-                if self.recid in self.cached_records:
-                    # Record has already been parsed, no need to add
-                    pass
-                else:
-                    # Add record to the global cache
-                    self.cached_records[self.recid] = self.cur_record
-                # Add record to the ordered list of results
-                self.records.append(self.cached_records[self.recid])
-
-            self.cur_controlfield.append(self.buffer)
-            self.in_controlfield = False
-            self.buffer = ""
-        elif name == "datafield":
-            self.in_datafield = False
-        elif name == "subfield":
-            self.in_subfield = False
-            self.cur_subfield.append(self.buffer)
-            self.buffer = ""
+"""DEPRECATED: use Invenio-Client package instead."""
 
+import warnings
 
-def decompose_code(code):
-    """
-    Decomposes a MARC "code" into tag, ind1, ind2, subcode
-    """
-    code = "%-6s" % code
-    ind1 = code[3:4]
-    if ind1 == " ": ind1 = "_"
-    ind2 = code[4:5]
-    if ind2 == " ": ind2 = "_"
-    subcode = code[5:6]
-    if subcode == " ": subcode = None
-    return (code[0:3], ind1, ind2, subcode)
+warnings.warn("Use Invenio-Client package instead.", DeprecationWarning)
```

Comment:
Since this is just a warning, should we still import from invenio-client and reexport here the corresponding functions?